oggm/core/models/flowline.py

```diff
@@ -300,6 +300,7 @@ def __init__(self, flowlines, mb_model=None, y0=0., fs=None, fd=None):
         """
 
         self.mb = mb_model
+        self.Aglen = Aglen
         self.fd = fd
```

Comment:
To preserve backwards compatibility (at least for a while), setting `fd` could be equivalent to setting `Aglen`, and the `self.fd` property could be removed. Something like:

```python
if fd is not None:
    Aglen = (N+2.) / 2
```

What do you think?